docs/source/technical_tutorials/dps_tutorial/DPS_runner_template.ipynb

```diff
@@ -0,0 +1,353 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "330e3ace",
+   "metadata": {},
+   "source": [
+    "# Prepare and launch a DPS batch of jobs for a particular algorithm\n",
+    "\n",
+    "**Goal**\n",
+    "Provide a template for DPS job submission which will be changed/adapted according to specific algorithms being run in DPS.\n",
+    "\n",
+    "**Motivation**  \n",
+    "It's easier to learn how to run many jobs of your script (where for each job there is some input that changes) if you can first see an example.\n",
+    "\n",
+    "Paul Montesano, PhD  \n",
+    "paul.m.montesano@nasa.gov  \n",
+    "June 2024"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 126,
+   "id": "ea7bcf9f",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from maap.maap import MAAP\n",
+    "maap = MAAP(maap_host='api.maap-project.org')"
```

Comment:
`maap = MAAP()`

Not sure if we want to encourage users providing a maap_host as it should default to the ops api.

Reply:
This was introduced as a fix to an issue with reading the maap.cfg file which has been resolved. https://github.com/MAAP-Project/Community/issues/909